docs/zh/faq/index.md

```
@@ -44,6 +44,18 @@ Swow 安装请参考 [Swow 官方文档](https://docs.toast.run/swow-blog/chs/in
 ## 上传图片或文件，访问Not Found 问题
 
 1. 生产环境下，建议使用nginx代理。
+
+  使用Nginx 代理可以借鉴以下配置 （注意 env 配置 和上传目录权限）。请注意，以下路径仅为示例，需根据实际部署环境调整。
+  假设资源url 为 https://example.com/uploads/**/****.png
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)